tests/e2e/src/eth_getBalance.test.js

```diff
@@ -0,0 +1,165 @@
+import goto from "./goto";
+import testDataManage from "./create_test_data/createTestDataManage";
+
+let testDataInfo = null;
+const pageName = "eth_getBalance.html";
+describe("eth_getBalance", () => {
+  testDataInfo = testDataManage.readTestDataAsJson("testData_1.json");
+  /**
+   * param1: real account address
+   * param2: real other number
+   */
+  it.skip("eth_getBalance_1", async () => {
+    await goto.goto(page, pageName);
+    const testType = await page.$(goto.pageIds.testTypeId);
+    const param1 = await page.$(goto.pageIds.param1Id);
+    const param2 = await page.$(goto.pageIds.param2Id);
```

Comment:
Please use Promise.all to run these tasks asynchronously for performance.

Reply:
Ref https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Reply:
Why not change here？

Reply:
will be changes in anther pr. 

> Why not change here？

will be change in another pr.